sys/src/cmd/frontusb/LICENSE

```diff
@@ -0,0 +1,20 @@
+Copyright © 2021 Plan 9 Foundation
+Copyright © 20XX 9front authors
```

Comment:
Time for some annoying license nit-picking: Since this comes from their boilerplate license file, it just says "20XX". I'm not sure what's exactly correct, but there should probably be a year in here. Maybe 2021, since we imported it in 2021? Otherwise you could look at the git history of the directory and see when they last modified it?